base/cvd/cuttlefish/host/libs/web/credential_source.cc

```diff
@@ -73,20 +73,22 @@ Result<std::unique_ptr<CredentialSource>> GetCredentialSourceLegacy(
   if (credential_source == "gce") {
     result = GceMetadataCredentialSource::Make(http_client);
   } else if (credential_source == "") {
-    LOG(VERBOSE) << "Probing oauth credentials at " << oauth_filepath;
     if (FileExists(oauth_filepath)) {
       std::string oauth_contents = CF_EXPECT(ReadFileContents(oauth_filepath));
       auto attempt_load = RefreshCredentialSource::FromOauth2ClientFile(
           http_client, oauth_contents);
       if (attempt_load.ok()) {
         result.reset(new RefreshCredentialSource(std::move(*attempt_load)));
+        LOG(INFO) << "\"" << oauth_filepath
+                  << "\" file found, using that as credentials";
       } else {
-        LOG(ERROR) << "Failed to load oauth2 credentials: "
-                   << attempt_load.error().FormatForEnv();
+        LOG(ERROR) << "Failed to load oauth credentials from \""
+                   << oauth_filepath
+                   << "\":" << attempt_load.error().FormatForEnv();
       }
     } else {
       LOG(INFO) << "\"" << oauth_filepath
-                << "\" missing, running without credentials";
+                << "\" file is missing, running without credentials";
```

Comment:
nit: I find `"/path/to/oauth/file/oauth.dat" file is missing` redundant, just `"/path....dat" is missing` sounds better IMHO.

Reply:
Done.  Also removed "file" file from found case.